HpToolsLauncher/MtbxManager.cs

```diff
@@ -150,7 +150,14 @@ public static List<TestInfo> LoadMtbx(string xmlContent, Dictionary<string, stri
                     if (xname != null)
                         name = xname.Value;
 
-                    TestInfo col = new TestInfo(path, name, testGroupName);
+                    // optional report path attribute
+                    XAttribute xReportPath = GetAttribute(test, "reportPath");
+                    string reportPath = null;
+
+                    if (xReportPath != null)
```

Comment:
I think there should be consistency between styling in one line if statements with the brackets or without; it's up to you